lib/scss_lint/linter/name_format.rb

```diff
@@ -39,7 +39,19 @@ def visit_variable(node)
       scaleX scaleY scaleZ
       skewX skewY
       translateX translateY translateZ
-      linear-gradient
+      linear-gradient repeating-linear-gradient
```

Comment:
Since you are adding sass specific functions and all we previously had was CSS functions, it might be nice to use a separate set for these to keep things clear.